console/src/main/resources/application.properties

```diff
@@ -28,17 +28,18 @@ server.port=8848
 # nacos.inetutils.ip-address=
 
 
+
 #*************** Config Module Related Configurations ***************#
-### If use MySQL as datasource:
+### If user MySQL as datasource:
```

Comment:
This file has modified by others. Please do not rollback it. I prefer that you checkout a new branch to do change.